packages/medusa/src/migrations/1666251508718-stock_location.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
```

Comment:
todo: remove this migration once https://github.com/medusajs/medusa/pull/2883 is merged